realm/src/main/java/io/realm/Realm.java

```diff
@@ -946,6 +946,30 @@ boolean contains(Class<? extends RealmObject> clazz) {
     }
 
     /**
+     * Return a distinct set of objects of a specific class. If no objects exist, the
```

Comment:
Lets put it this way: I should have known better and I wondered. Having a line like "As a Realm is unordered, it is undefined which element is returned for values with multiple candidates"  (could probably be rephrased)

It makes the contract of the method totally clear and might save a support ticket :)